modules/gitops-argo-cd-properties.adoc

```diff
@@ -16,160 +16,162 @@ The Argo CD Custom Resource consists of the following properties:
-  * `<sharding.maxShards>` - The maximum number of Argo CD Application Controller replicas.
-  * `<sharding.clustersPerShard>` - The number of clusters that need to be managed by each shard. When the replica count reaches the `maxShards`, the shards manage more than one cluster.
-  * `<env>` - Environment to set for the application controller workloads.
-  * `<sourceNamespaces>` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
-  * `<extraCommandArgs>` - List of arguments added to the existing arguments set by the Operator.
+  * `scmRootCAConfigMap` - The name of the config map that stores the Gitlab SCM Provider's TLS certificate that will be mounted on the Application Set Controller at the "/app/tls/scm/cert" path.
+  * `webhookServer` - Defines the available options for the ApplicationSet webhook server.
+
+|`banner` |Adds a UI banner message.|`__object__`
+a|* `banner.content` - The banner message content. This content is required if a banner is displayed.
+  * `banner.url` - An optional banner message link URL.
+
+|`configManagementPlugins`    |Adds a configuration management plugin.| `__empty__` |
+
+|`controller`    |Argo CD Application Controller options.| `__object__`
+a|* `processors.operation` - The number of operation processors.
+  * `processors.status` - The number of status processors.
+  * `resources` - The container compute resources.
+  * `logLevel` - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
+  * `appSync` - AppSync is used to control the sync frequency of Argo CD applications.
+  * `sharding.enabled` - Enable sharding on the Argo CD Application Controller component. Use this property to manage a large number of clusters and relieve memory pressure on the controller component.
+  * `sharding.replicas` - The number of replicas that are used to support sharding of the Argo CD Application Controller.
+  * `sharding.dynamicScalingEnabled` - Enables the dynamic scaling of the Argo CD Application Controller component. Use this property if you want the Operator to scale the number of replicas based on the number of clusters the controller component is managing. If you set this property to `true`, it overrides the configuration of the `sharding.enabled` and `sharding.replicas` properties.
+  * `sharding.minShards` - The minimum number of Argo CD Application Controller replicas.
+  * `sharding.maxShards` - The maximum number of Argo CD Application Controller replicas.
+  * `sharding.clustersPerShard` - The number of clusters that need to be managed by each shard. When the replica count reaches the `maxShards`, the shards manage more than one cluster.
+  * `env` - Environment to set for the application controller workloads.
+  * `sourceNamespaces` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
```

Comment:
@Dhruv-Soni11 please remove this field from .spec.controller

cc: @svghadi

Reply:
Fixed.

Reply:
Fixed.